src/overlays/actors/ovl_En_Rr/z_en_rr.h

```diff
@@ -6,9 +6,57 @@
 
 struct EnRr;
 
+typedef void (*EnRrActionFunc)(struct EnRr*, GlobalContext*);
+
+typedef struct EnRrBodySegment {
```

Comment:
```suggestion
typedef struct {
```

Reply:
Why do we prefer anonymous structs?

Reply:
As long as we can, we use types for structs, not the struct itself (ie. we use them as `EnRrBodySegment`, not `struct EnRrBodySegment`). In this case the struct name isn't needed so we can just omit it.